internal/git/branch.go

```diff
@@ -20,9 +20,26 @@ type LocalBranch struct {
 	CheckedOut bool
 }
 
+// LocalBranchesOptions specifies options for listing local branches.
+type LocalBranchesOptions struct {
+	// Sort specifies an optional value for git branch --sort.
+	Sort string
+}
+
 // LocalBranches lists local branches in the repository.
-func (r *Repository) LocalBranches(ctx context.Context) ([]LocalBranch, error) {
-	cmd := r.gitCmd(ctx, "branch")
+func (r *Repository) LocalBranches(ctx context.Context, opts *LocalBranchesOptions) ([]LocalBranch, error) {
+	var sort string
+	if opts != nil {
+		sort = opts.Sort
+	}
+
+	var cmd *gitCmd
+	if sort != "" {
+		cmd = r.gitCmd(ctx, "branch", "--sort="+sort)
+	} else {
+		cmd = r.gitCmd(ctx, "branch")
+	}
```

Comment:
nit: We can compile the list of arguments instead:

```suggestion
	args := []string{"branch"}
	if opts.Sort != "" {
		args = append(args, "--sort="+opts.Sort)
	}
	cmd := r.gitCmd(ctx, args...)
```